tests/evaluator_tests.py

```diff
@@ -57,12 +57,12 @@ def runTest(self):
         total_ratings = numpy.count_nonzero(self.ratings_matrix)
 
         # ensure that each fold has 1/k of the total ratings
-        k_inverse = (1 / self.k_folds)
-        self.assertEqual(k_inverse, numpy.count_nonzero(test1) / total_ratings)
+        k_inverse = '%.1f' % numpy.round((1 / self.k_folds), 2)
+        self.assertEqual(k_inverse, '%.1f' % numpy.round(numpy.count_nonzero(test1) / total_ratings, 1))
```

Comment:
For these 3 assertions, to assert that 2 floats a and b are equal (or almost equal) it's better to make it like assertTrue(abs(a - b) < EPS) # Where EPS is a small value like 1e-6 (I used this usually) or 1e-8
it's a bit hacky to compare the string representations.